Client/MoonlightClient.cpp

```diff
@@ -66,10 +68,13 @@ int MoonlightClient::StartStreaming(std::shared_ptr<DX::DeviceResources> res,Str
 	callbacks.stageFailed = stage_failed;
 	callbacks.stageComplete = connection_status_update;
 	callbacks.rumble = connection_rumble;
-	FFMpegDecoder::createDecoderInstance(res, this->pacer);
+	FFMpegDecoder::createDecoderInstance(res);
 	DECODER_RENDERER_CALLBACKS rCallbacks = FFMpegDecoder::getDecoder();
 	AUDIO_RENDERER_CALLBACKS aCallbacks = AudioPlayer::getDecoder();
-	int k = LiStartConnection(&serverData.serverInfo, &config, &callbacks, &rCallbacks, &aCallbacks, NULL, 0, NULL, 0);
+	auto gamepads = Windows::Gaming::Input::Gamepad::Gamepads;
+	//Since we are on Xbox, we can assume at least one gamepad is connected since they are required on this platform
+	this->SetGamepadCount(max(1, gamepads->Size));
+	int k = LiStartConnection(&serverData.serverInfo, &config, &callbacks, &rCallbacks, &aCallbacks, NULL, 0, NULL, activeGamepadMask);
```

Comment:
the last parameter of `LiStartConnection()` is for audio renderer flags, not the gamepad mask.